scripts/get_files_on_gcs.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
this script is pretty bespoke. can we move to marin?

Reply:
Moved to https://github.com/stanford-crfm/marin/pull/40